data-pipeline-ffi/src/trace_exporter.rs

```diff
@@ -282,13 +265,13 @@ pub unsafe extern "C" fn ddog_trace_exporter_free(handle: Box<TraceExporter>) {
 ///   TraceExporter. The memory for the trace must be valid for the life of the call to this
 ///   function.
 /// * `trace_count` - The number of traces to send to the Datadog Agent.
-/// * `response` - Optional parameter that will ontain the agent response information.
+/// * `response_out` - Optional handle to store a pointer to the agent response information.
```

Comment:
Just curious - is `_out` standard convention for input parameters that are pointers to output?